doc/tutorials/isingmodel.ipynb

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
<p>Let's test these functions for the [1,-1,-1] spin configuration. </p><p>The total energy for this Ising model should be </p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>